python/pyspark/mllib/classification.py

```diff
@@ -20,12 +20,10 @@
 import warnings
 
 import numpy
-from numpy import array
```

Comment:
Hm, right, so this looks false positive. It's passed via module's namespaces via `__dict__`. when it runs doc test. 

This could be fixed:

1. fix it within `def _test`

```python
from numpy import array
globs['array'] = array
```

2. Explicitly import NumPy's `array` within each doctest - `array` in the doctest confusing between `array.array` and `numpy.array` anyway.

Reply:
Sean, I made a PR to your branch while I'm reviewing and testing (https://github.com/srowen/spark/pull/5) Feel free to use it :D.

Reply:
Right, I was watching out for false positives, but missed this one. We should just leave in imports that doctests use, but in this case I agree it's easier to clarify with `numpy.array`. Was this the only one, from your PR? I'll look over the array import changes too.

Reply:
Yea, I think I spot all of them. I doubly checked but .. test results will tell us :).